packages/sitecore-jss/src/graphql/index.ts

```diff
@@ -1,7 +1,9 @@
 export { getAppRootId, AppRootQueryResult } from './app-root-query';
+export { ClientError } from 'graphql-request';
```

Comment:
We don't want to export `ClientError` here, right (they should use `GraphQLClientError`)?

Reply:
Ohh yeah right.